1907-08-10.xml

```diff
@@ -935,6 +896,7 @@
                     <p>J. Margosches</p>
                     <p>Cairo Bulac Road</p>
                 </div>
+                <div type="item" status="verified"></div>
```

Comment:
The items on this page are not verified (i.e., correct) if the ad template is still missing.